css/style.css

```
@@ -0,0 +1,505 @@
+/* Fonts */
+
+@font-face {
+  font-family: "Cuprum";
+  src: url(../fonts/cuprum.woff) format("woff"),
+    url(../fonts/cuprum.woff2) format("woff2");
+  font-weight: 400;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Cuprum";
+  src: url(../fonts/cuprumbold.woff) format("woff"),
+    url(../fonts/cuprumbold.woff2) format("woff2");
+  font-weight: 700;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Cuprum";
+  src: url(../fonts/cuprumbolditalic.woff) format("woff"),
+    url(../fonts/cuprumbolditalic.woff2) format("woff2");
+  font-weight: 700;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: "PT Sans";
+  src: url(../fonts/ptsans.woff) format("woff"),
+    url(../fonts/ptsans.woff2) format("woff2");
+  font-weight: 400;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "PT Sans";
+  src: url(../fonts/ptsansbold.woff) format("woff"),
+    url(../fonts/ptsansbold.woff2) format("woff2");
+  font-weight: 700;
+  font-style: normal;
+}
+
+/* Variables */
+
+:root {
+  /* Index */
+
+  --basic-color:#fff;
```

```suggestion
  --basic-color: #FFFFFF;
```
Это вопрос единообразия оформления кода. Сокращённую запись не используем и поскольку для остальных цветов вы используете запись в верхнем регистре, то нужно так делать везде.